XenAdmin/TabPages/SnapshotsPage.cs

```diff
@@ -119,7 +127,8 @@ public VM VM
                     m_VM.PropertyChanged += snapshot_PropertyChanged;
                     //Version setup
                     toolStripMenuItemScheduledSnapshots.Available = toolStripSeparatorView.Available = !Helpers.FeatureForbidden(VM.Connection, Host.RestrictVMSnapshotSchedule);
-                    if (VM.SnapshotView != SnapshotsView.ListView)
+
+                    if (_storedViewsPerVm.ContainsKey(m_VM.opaque_ref) && _storedViewsPerVm[m_VM.opaque_ref] != SnapshotsView.ListView)

```

Comment:
Changed behaviour: Listview is now the default, because `_storedViewsPerVm.ContainsKey(m_VM.opaque_ref)` is always false if I look the first time at snapshot after XenCenter start.

https://github.com/xcp-ng/xenadmin/issues/62